Pyjion/absint.cpp

```diff
@@ -1319,8 +1327,29 @@ void AbstractInterpreter::branch_raise(char *reason) {
             break;
         }
     }
+    return count;
+}
+
+void AbstractInterpreter::ensure_labels(vector<Label>& labels, size_t count) {
+    for (auto i = labels.size(); i < count; i++) {
+        labels.push_back(m_comp->emit_define_label());
+    }
+}
+
+void AbstractInterpreter::branch_raise(char *reason) {
+    auto ehBlock = get_ehblock();
```

Comment:
The variable name doesn't seem to follow the general style we have been going with (PEP 7?).